### PR TITLE
Update protobuf to 27.2 & rebuilds

### PR DESCRIPTION
**Pull request by joebonrichie**
**Summary**

Update protobuf to 27.2 and perform rebuilds

**Test Plan**

Add as local repo, upgrade, verify any updated pkgs were okay

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Review by EbonJaeger — DISMISSED** [review]
LGTM, thanks!

File: packages/c/curaengine/package.yml [hunk 1]
```diff
@@ -34,12 +34,12 @@ setup      : |
     # Remove bundled libraries
     rm -rf libs
 
-    %cmake_ninja -DSET_RPATH:BOOL=OFF \
```
Comment (ReillyBrogan) [inline packages/c/curaengine/package.yml:37]: But why

File: packages/p/protobuf/pspec_x86_64.xml [hunk 2]
```diff
@@ -21,10 +21,16 @@
...
-            <Path fileType="executable">/usr/bin/protoc-25.3.0</Path>
-            <Path fileType="library">/usr/lib64/libprotobuf-lite.so.25.3.0</Path>
-            <Path fileType="library">/usr/lib64/libprotobuf.so.25.3.0</Path>
-            <Path fileType="library">/usr/lib64/libprotoc.so.25.3.0</Path>
+            <Path fileType="executable">/usr/bin/protoc-27.2.0</Path>
+            <Path fileType="executable">/usr/bin/protoc-gen-upb</Path>
+            <Path fileType="executable">/usr/bin/protoc-gen-upb-27.2.0</Path>
+            <Path fileType="executable">/usr/bin/protoc-gen-upb_minitable</Path>
+            <Path fileType="executable">/usr/bin/protoc-gen-upb_minitable-27.2.0</Path>
+            <Path fileType="executable">/usr/bin/protoc-gen-upbdefs</Path>
```
Comment (ReillyBrogan) [inline packages/p/protobuf/pspec_x86_64.xml:29]: Some of these binaries seem like they should be in devel instead

File: packages/c/curaengine/package.yml (hunk 1 above)
Comment (joebonrichie) [reply packages/c/curaengine/package.yml:37]: buggy cmake_ninja macro

```
+ ninja install -j16 -C solusBuildDir
ninja: Entering directory `solusBuildDir'
[0/1] Install the project...
/bin/sh: line 1: cd: /home/build/YPKG/root/curaengine/build/CuraEngine-5.4.0/solusBuildDir: Not a directory
FAILED: CMakeFiles/install.util 
cd /home/build/YPKG/root/curaengine/build/CuraEngine-5.4.0/solusBuildDir && /usr/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.
```

Comment (joebonrichie) [reply packages/c/curaengine/package.yml:37]: same for libarcus. This happens sometimes and i don't know why

**Review by ReillyBrogan — APPROVED** [review]

**Pull request merged**